src/java.base/share/classes/java/security/spec/EncodedKeySpec.java

```diff
@@ -74,11 +74,11 @@ public EncodedKeySpec(byte[] encodedKey) {
      *
      * @param encodedKey the encoded key. The contents of the
      * array are copied to protect against subsequent modification.
-     * @param algorithm the algorithm name of the encoded key
-     * See the KeyFactory section in the <a href=
-     * "{@docRoot}/../specs/security/standard-names.html#keyfactory-algorithms">
+     * @param algorithm the algorithm name of the encoded key.
+     * See the AsymmetricKey Algorithms section in the
+     * <a href="{@docRoot}/../specs/security/standard-names.html#asymmetrickey-algorithms">
      * Java Security Standard Algorithm Names Specification</a>
-     * for information about standard algorithm names.
+     * for information about standard asymmetric key algorithm names.
```

Comment:
Suggest adding an @spec line as in `Key.getAlgorithm()`. Same comment applies to other changes.